aiohttp.md

```diff
@@ -0,0 +1,31 @@
+# 24-08-2021
+
+## Aiohttp Client settings
+
+```python
+import aiohttp
+
+
+class HttpClient:
+    session: aiohttp.ClientSession = None
+
+    def start(self):
+        self.session = aiohttp.ClientSession(
+            connector=aiohttp.TCPConnector(force_close=True),
+            timeout=aiohttp.ClientTimeout(total=5)
+        )
+
+    async def stop(self):
+        await self.session.close()
+        self.session = None
+
+    def __call__(self) -> aiohttp.ClientSession:
+        assert self.session is not None
+        return self.session
+```
+
+1. Having a singleton like `HttpClient`, can be used in conjucntion with your web server that makes 3rd party API call. Only initialize the session once at the beginning of the startup
```

Comment:
`conjunction`?

Reply:
I am curious what are the pros and cons of doing so

Reply:
not super sure. but initializing objects has a takes more than zero